### DISCUSSION TITLE
Average download and upload speed

### DISCUSSION THREAD

**Discussion by Ademandi** [alexjustesen/speedtest-tracker#1330] [Ideas]
Hi all,

first of all I want to mention, that this software i amazing. The desing as well as the functionality is awesome and a great substitute for the tool from henrywhitaker (https://github.com/henrywhitaker3/Speedtest-Tracker).

One feature that I am missing a bit is the average of down- and upload speeds that are recorded.
That would make the software complete in my opinion.

**Comment by alexjustesen** [comment]
Dup of #1244

**Reply by Ademandi** [reply]
Cool. Thanks a lot
